proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
Don't specify the `https-port` :-) We have a working example on the linked [blog post here](https://github.com/jpkrohling/hawkular.github.io/blob/JPK-JaegerWithSecurityProxy/src/main/jbake/content/blog/2017/07/jaeger-with-security-proxy.adoc#preparing-the-jaeger-openshift-template) , under the "ConfigMap" section.